rllib/env/single_agent_episode.py

```diff
@@ -143,40 +141,39 @@ def concat_episode(self, episode_chunk: "SingleAgentEpisode"):
 
         # Extend ourselves. In case, episode_chunk is already terminated (and numpyfied)
         # we need to convert to lists (as we are ourselves still filling up lists).
-        self.observations.extend(list(episode_chunk.observations))
-        self.actions.extend(list(episode_chunk.actions))
-        self.rewards.extend(list(episode_chunk.rewards))
-        self.infos.extend(list(episode_chunk.infos))
+        self.observations.extend(episode_chunk.observations)
+        self.actions.extend(episode_chunk.actions)
+        self.rewards.extend(episode_chunk.rewards)
+        self.infos.extend(episode_chunk.infos)
         self.t = episode_chunk.t
-        self.states = episode_chunk.states
 
         if episode_chunk.is_terminated:
             self.is_terminated = True
         elif episode_chunk.is_truncated:
             self.is_truncated = True
 
         for k, v in episode_chunk.extra_model_outputs.items():
-            self.extra_model_outputs[k].extend(list(v))
+            self.extra_model_outputs[k].extend(v)
 
         # Validate.
         self.validate()
 
-    def add_initial_observation(
+    def add_env_reset(
```

Comment:
Changed name for clarity:
1) `add_env_reset()`: Add all the data returned by an env.reset call
2) `add_env_step()`: Add all the data returned by an env.step call